lib/sparql_to_sw_solr/instance_solr_doc.rb

```diff
@@ -58,6 +61,7 @@ def instance_uri_to_ckey
     def values_from_solutions(solutions, predicate_name)
       values = []
       solutions.each_solution do |soln|
+        next unless soln.bindings.keys.include?(:o) && soln.bindings.keys.include?(:p)
```

Comment:
it fails specs without it - it's so the specs don't fail when we're mocking an RDF::Solutions with different bindings